### DISCUSSION TITLE
Trying to get :after and hover::after work

### DISCUSSION THREAD

**Discussion by Devotee007** [tailwindlabs/tailwindcss#6035] [Help] [answered]
Hi,

I try to do an underline that animates on hover. But I can't get it to work. When I test the code below I don't see it when I inspect the code in Chrome, the :after isn't there. What am I doing wrong? :)  And then I want it to change on hover, but can't find information on how to user ::after on hover?

`<a href="#" class="inline-block no-underline after: block w-0 h-0.5 bg-black transition-all duration-300 `">Lorem` ipsum</a>`

EDIT:

I think I have figured out now that I have to put it like after:block after:w-0 after:h-0.5 and so on, but it still doesn't work. 

//Devotee

**Comment by olets** [comment, marked as answer]
In your example there are no `hover:after:` styles. You haven't said what your goal is, but my guess is you're missing `hover:after:w-full`.

https://play.tailwindcss.com/u7ctXP3sMF

**Reply by Devotee007** [reply]
Thanks for your help! That's exactly what I tried to do on :after. But when I try the code you linked to on my page it doesn't work. Do I have to add :after under variants in the config file or do something else?

**Reply by Devotee007** [reply]
I started JIT and now it works, I thought it would work without enabling JIT.
